markdown/_index.md

```diff
@@ -80,9 +81,9 @@ Siehe auch `["Hinweise zum Praktikum"]({{< ref "/org/grading" >}})`{=markdown}
 
 ### Tutorium: 1 SWS
 
--   Mo, 13:30 -- 15:00 Uhr (online)
+-   Mo, 13:30 -- 15:00 Uhr (J107)
 
-**Anmerkung**: Das Tutorium in PM ist fester Bestandteil der Prüfungsordnung.
+**Anmerkung**: Das Tutorium in PM ist fester Bestandteil der Prüfungsordnung. Es besteht Teilnahmepflicht.
```

Comment:
okay?!

Reply:
tja, wenn sie sich beschweren, dann kriegen sie soviel präsenz wie möglich. streng genommen ist das 1 sws, welches zu den credits gehört. ... aber ja, in der orga-runde heute (waaaaaahnsinn, immer noch vieeeel zu komplex) kam da mehr als eine frage und mein herumlavieren sollte eigentlich bände gesprochen haben ...